res/skins/LateNight/skin.xml

```diff
@@ -39,9 +39,12 @@
     <attributes>
       <!-- Set skin defaults. -->
       <attribute config_key="[Master],num_decks">4</attribute>
+      <attribute config_key="[Master],num_samplers">8</attribute>
       <attribute persist="true" config_key="[Microphone],show_microphone">0</attribute>
       <attribute persist="true" config_key="[EffectRack1],show">0</attribute>
       <attribute persist="true" config_key="[Samplers],show_samplers">0</attribute>
+      <attribute persist="true" config_key="[Master],show_4sampler">0</attribute>
```

Comment:
By "8 samplers are default" I meant to use a CO like [Skin],8_samplers and set it to 1.
This is consistent with [..],show_4deck, [..],show_4effectunits & [..],8_hotcues

Reply:
ok, done